gears/src/types/base/coins/decimal.rs

```diff
@@ -20,6 +20,10 @@ impl DecimalCoins {
         }
 
         for coin in other {
+            // Note:
+            // operator '>' means that all values are greater or equal
```

Comment:
I find these statements a bit confusing. Are they referring to the `is_all_gte` method?

Reply:
Yes, it refers to method `is_all_gte` that means all values of another coins are greater or equal 
```rs
            if coin.amount > self.amount_of(&coin.denom) {
                return false;
            }
```

This code means that method returns false for exactly greater values of other coins. 
For example;
```rs 
let a = vec![2,3,4];
let b = vec![1, 3, 2]; 
assert!(a.is_all_gte(b));
assert!(!a.is_all_gt(b));

let a = vec![2,3,4];
let b = vec![1, 3, 5]; 
assert!(!a.is_all_gte(b));
assert!(!a.is_all_gt(b));
```